python/ray/ml/train/integrations/sklearn/sklearn_trainer.py

```diff
@@ -112,26 +152,272 @@ def __init__(
         estimator: BaseEstimator,
         datasets: Dict[str, GenDataset],
         label_column: Optional[str] = None,
+        params: Optional[Dict[str, Any]] = None,
         scoring: Optional[ScoringType] = None,
         cv: Optional[CVType] = None,
+        parallelise_cv: Optional[bool] = None,
+        return_train_score_cv: bool = False,
         scaling_config: Optional[ScalingConfig] = None,
         run_config: Optional[RunConfig] = None,
         preprocessor: Optional[Preprocessor] = None,
-        resume_from_checkpoint: Optional[Checkpoint] = None,
```

Comment:
Can we still check if this is set below? I.e. `if fit_params.pop("resume_from_checkpoint", None): raise ValueError("SklearnTrainer does not support resuming from checkpoints...")`